src/main/java/io/confluent/connect/jdbc/dialect/GenericDatabaseDialect.java

```diff
@@ -1246,12 +1246,14 @@ protected ColumnConverter columnConverterFor(
 
       // Date is day + month + year
       case Types.DATE: {
-        return rs -> rs.getDate(col, DateTimeUtils.getTimeZoneCalendar(timeZone));
+        return rs -> rs.getDate(col,
+            DateTimeUtils.getTimeZoneCalendar(TimeZone.getTimeZone(ZoneOffset.UTC)));
       }
 
       // Time is a time of day -- hour, minute, seconds, nanoseconds
       case Types.TIME: {
-        return rs -> rs.getTime(col, DateTimeUtils.getTimeZoneCalendar(timeZone));
+        return rs -> rs.getTime(col,
+            DateTimeUtils.getTimeZoneCalendar(TimeZone.getTimeZone(ZoneOffset.UTC)));
```

Comment:
I think this change is explicitly incorrect change, because interpreting times as the specified timezone is an explicitly supported functionality of this connector.

Reply:
Thinking about this more, I actually think this change may be a no-op. For Dates, the timestamp of start of day differs depending on timezone. But, times are just millis since EPOCH and what timezone we ask for them in should not make a difference. I can back out the changes for Time and just keep the one for Date.